<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btnFormKaryawan">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>FORM KARYAWAN</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>APLIKASI PENGGAJIAN</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnFormBagian">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>110</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>FORM BAGIAN</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnFormAbsensi">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>170</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>FORM ABSENSI</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnFormLembur">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>230</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>FORM LEMBUR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnFormPenggajian">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>290</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>FORM PENGGAJIAN</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
